pkg/server/conn.go

```diff
@@ -1212,7 +1212,7 @@ func (cc *clientConn) Run(ctx context.Context) {
 						timestamp = ctx.GetSessionVars().TxnCtx.StaleReadTs
 					}
 				}
-				logutil.Logger(ctx).Info("command dispatched failed",
+				logutil.Logger(ctx).Warn("command dispatched failed",
```

Comment:
Should we consider handling `ErrKeyExists` as a noisy / expected error (similar to `errIsNoisy`) and filter it out?
Otherwise, this could result in a large number of warning logs in certain scenarios, which may reduce log signal quality.